client/public/index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="utf-8">
     <title>Follow Me</title>
+    <input type="button" onclick="window.location.href='/logout';" value="Logout" />
```

Comment:
This doesn't look right. Don't think you should be putting an input tag in the head...